go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/BurntSushi/toml v1.1.0
 	github.com/celestiaorg/celestia-app v0.5.2
 	github.com/celestiaorg/go-libp2p-messenger v0.1.0
-	github.com/celestiaorg/nmt v0.8.0
+	github.com/celestiaorg/nmt v0.10.0
```

Comment:
cc @evan-forbes

Reply:
That is also sth the node team should consider. cc @Wondertan @renaynay

As you already depend on app, you could use the re-exported types, which would make nmt and maybe rsmt2d indirect deps instead of direct deps. Dependabot wouldn't bother you about nmt, rsmt2d independently. Just sth to consider.

Reply:
It might be a good idea for the app to re-export this. Not sure yet. Resolving this as it fine to depend on 0.10.0 fo now